src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml

```diff
@@ -42,93 +38,79 @@
             <RowDefinition Height="40" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
-        <Grid Name="SearchText"
-              Grid.Row="0">
+        <Grid Name="SearchText" Grid.Row="0">
             <Grid.ColumnDefinitions>
                 <ColumnDefinition Width="Auto" />
                 <ColumnDefinition Width="*" />
             </Grid.ColumnDefinitions>
-            <Border CornerRadius="8"
-                    Background="{StaticResource SearchTextBoxBackground}"
-                    BorderBrush="{StaticResource outerBorderSelection}"
-                    BorderThickness="1"
+            <Border Grid.ColumnSpan="2"
+                    Margin="0,0,0,2"
                     Padding="5"
-                    Margin="1 3"
-                    Grid.ColumnSpan="2">
+                    Background="{StaticResource SearchTextBoxBackground}"
+                    BorderBrush="#2A576C"
+                    BorderThickness="0,0,0,1">
                 <Grid>
-                    <StackPanel Name="SearchIconAndTextBlockGrid"
-                                Grid.ColumnSpan="2"
-                                Width="Auto"
-                                Orientation="Horizontal">
+                    <DockPanel Name="SearchIconAndTextBlockGrid" Grid.ColumnSpan="2">
                         <Image x:Name="SearchIcon"
-                               Width="18"
-                               Height="18"
+                               Width="20px"
+                               Height="20px"
+                               HorizontalAlignment="Left"
                                VerticalAlignment="Center"
-                               HorizontalAlignment="Left">
+                               DockPanel.Dock="Right">
                             <Image.Style>
                                 <Style TargetType="{x:Type Image}">
-                                    <Setter Property="Source"
-                                            Value="/DynamoCoreWpf;component/UI/Images/search_normal.png" />
```

Comment:
If we are no longer using such resource image, should we remove it?

Reply:
I think it is still being used in node autocompletion window: https://github.com/DynamoDS/Dynamo/blob/d9e2186364d66badcf07f514d2bfae91772e91f7/src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml#L176 and also the search icon in library view extension. We should change them as well. Do we want it do it in a different PR or this one?

Reply:
Good points @reddyashish should be in different PRs.